scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala

```diff
@@ -79,14 +79,21 @@ private[types] object TypeProvider {
     checkMacroEnclosed(c)
 
     val (r, caseClassTree, name) = annottees.map(_.tree) match {
-      case List(q"case class $name(..$fields) { ..$body }") =>
+      case l @ List(q"$mods class $name[..$_] $_(..$fields) extends { ..$_ } with ..$_ { $_ => ..$body }") if mods.asInstanceOf[Modifiers].hasFlag(Flag.CASE) =>
+
+        val desc = l.head.asInstanceOf[ClassDef].mods.annotations
+          .filter(_.children.head.toString() == "new description")
```

Comment:
Can't seem to find a better way to get this - `tpe` for each child is `null` :( any ideas here?